source/features/clear-pr-merge-commit-message.tsx

```diff
@@ -23,7 +22,8 @@ async function clear(messageField: HTMLTextAreaElement): Promise<void | false> {
 		return false;
 	}
 
-	set(messageField, cleanedMessage ? cleanedMessage + '\n' : '');
+	// Do not use `text-field-edit` #6348
+	messageField.value = cleanedMessage ? cleanedMessage + '\n' : '';
```

Comment:
The advantages `text-field-edit` provides (undoability) aren't worth the issues in this important part of the UI.

- https://github.com/fregante/text-field-edit/issues/21